shardingsphere-proxy/shardingsphere-proxy-common/src/main/java/org/apache/shardingsphere/proxy/config/ProxyConfigurationLoader.java

```diff
@@ -64,9 +67,12 @@ public static YamlProxyConfiguration load(final String path) throws IOException
                 YamlProxyRuleConfiguration::getSchemaName, each -> each, (oldValue, currentValue) -> oldValue, LinkedHashMap::new)));
     }
     
-    private static File getResourceFile(final String path) {
+    private static File getResourceFile(final String path) throws IOException {
         URL url = ProxyConfigurationLoader.class.getResource(path);
-        return null == url ? new File(path) : new File(url.getFile());
+        if (null != url) {
```

Comment:
return new File(URLDecoder.decode(null != url ? url.getFile() : path, "UTF-8"));

will be better.

Reply:
I agree